devices/org.osbuild.lvm2.lv

```diff
@@ -182,6 +186,12 @@ class LVService(devices.DeviceService):
 
         # Find the volume group that belongs to the device specified
         # via `parent`
+        # XXX: does parent_path already needs changing above for
+        # manage_devices_files?
+        # XXX2: add test
+        part = options.get("vg_partnum")
```

Comment:
There is also the open question if this should be called `vg_partition` or `vg_partnum` - the original name was `vg_partition` but because of `partnum` is used by partscan so I renamed it. Now looking at this again I'm not so sure anymore, we also use "partition" in "mounts" so it's probably best to change it back. Sorry! (I do like partnum slightly better than partition though as it makes it clear it's really just the number *so* maybe worth the bikeshed)

Reply:
I prefer `vg_partnum` too.  Don't put a lot of weight on the original name I used in the comments in the previous PR; there was no thought put into the naming there. :)